package.json

```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/ArrowSphere/nodejs-api-client.git"
   },
-  "version": "3.90.0",
+  "version": "3.91.0",
```

Comment:
This change should be in a different commit named like this:
**chore(package): version 3.91.0**

You can do a git log if you want to check